packages/ReflectionDocBlock/tests/NodeAnalyzer/DocBlockAnalyzerTest.php

```diff
@@ -30,6 +30,8 @@ public function testHasAnnotation(): void
     public function testRemoveAnnotationFromNode(): void
     {
         $node = $this->createNodeWithDoc('@param ParamType $paramName');
+        assert($node->getDocComment() !== null);
+
```

Comment:
Why not PHPUnit assert?

Reply:
We can add the `assertNotNull`, but this `assert` is needed it to fix PHPStan :(

Reply:
Well, the method above only returns node with doc comment, so the code here is correct.

That's fail positive by PHPStan. It should not pollute code just to make it silent on places where it can't detect correctly. And it's completely fine, I don't assume to know it all.
That's what `parameters > ignoreErrors` are for.

Reply:
Yes, I just remove some ignored error as we have 100+, and IMHO isn't healthy to our code. I'll try to remove this `assert`.

Reply:
> Well, the method above only returns node with doc comment, so the code here is correct.

Actually, it can return `null`, just checked: https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/NodeAbstract.php#L115

I'm pushing a new commit with `assertNotNull`, but we must use `assert` here. Otherwise, is a test class, is good to do it :)

Reply:
I mean the code line above:

```php
$this->createNodeWithDoc('@param ParamType $paramName');
```

Reply:
> Yes, I just remove some ignored error as we have 100+, and IMHO isn't healthy to our code. 

Why do you think number of ignore errors is related to code quality?
Well, it should be called "ignored reports", since it's often PHPStan errors.